src/renderers/dom/shared/__tests__/CSSPropertyOperations-test.js

```diff
@@ -16,97 +16,47 @@ var ReactDOM = require('react-dom');
 var ReactDOMServer = require('react-dom/server');
 
 describe('CSSPropertyOperations', () => {
-  var CSSPropertyOperations;
-
-  beforeEach(() => {
-    jest.resetModules();
-    CSSPropertyOperations = require('CSSPropertyOperations');
-  });
-
-  it('should create markup for simple styles', () => {
-    expect(
-      CSSPropertyOperations.createMarkupForStyles({
-        backgroundColor: '#3b5998',
-        display: 'none',
-      }),
-    ).toBe('background-color:#3b5998;display:none');
-  });
-
-  it('should ignore undefined styles', () => {
-    expect(
-      CSSPropertyOperations.createMarkupForStyles({
-        backgroundColor: undefined,
-        display: 'none',
-      }),
-    ).toBe('display:none');
-  });
-
-  it('should ignore null styles', () => {
-    expect(
-      CSSPropertyOperations.createMarkupForStyles({
-        backgroundColor: null,
-        display: 'none',
-      }),
-    ).toBe('display:none');
-  });
-
-  it('should return null for no styles', () => {
-    expect(
-      CSSPropertyOperations.createMarkupForStyles({
-        backgroundColor: null,
-        display: null,
-      }),
-    ).toBe(null);
-  });
-
   it('should automatically append `px` to relevant styles', () => {
-    expect(
-      CSSPropertyOperations.createMarkupForStyles({
-        left: 0,
-        margin: 16,
-        opacity: 0.5,
-        padding: '4px',
-      }),
-    ).toBe('left:0;margin:16px;opacity:0.5;padding:4px');
+    var styles = {
```

Comment:
I left a few tests here that specifically assert things about markup (e.g. that it doesn't have extra `px` or spaces).
Those that only assert correctness are replaced with integration tests.